product-mini/platforms/nuttx/CMakeLists.txt

```diff
@@ -49,6 +49,12 @@ else()
   add_definitions(-DWASM_ENABLE_WORD_ALIGN_READ=0)
 endif()
 
+if(CONFIG_INTERPRETERS_WAMR_AOT_DEBUG)
+  add_definitions(-DWASM_ENABLE_DYNAMIC_AOT_DEBUG=1)
```

Comment:
This should add to the common CMake build system, not only NuttX's

Reply:
Yes, had better add lines in `build-scripts/config_common.cmake`, like:
```cmake
if (WAMR_BUILD_DYNAMIC_AOT_DEBUG EQUAL 1)
  add_definitions (-DWASM_ENABLE_DYNAMIC_AOT_DEBUG=1)
  message ("     Dynamic AOT debug enabled")
endif ()
```

Reply:
done